### PR TITLE
New package: Araxis.Merge version 2024.5981

### PR DESCRIPTION
**Pull request by johnburnett**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/149681)

### CODE REVIEW

**Comment by stephengillie** [comment]
Verify: 
image

File: manifests/a/Araxis/Merge/2024.5981/Araxis.Merge.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Araxis.Merge
+PackageVersion: "2024.5981"
```
Comment (stephengillie) [inline manifests/a/Araxis/Merge/2024.5981/Araxis.Merge.installer.yaml:5]:
```suggestion
PackageVersion: 70.30.5981
```
Please update file name and path to match this change.

File: manifests/a/Araxis/Merge/2024.5981/Araxis.Merge.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Araxis.Merge
+PackageVersion: "2024.5981"
```
Comment (stephengillie) [inline manifests/a/Araxis/Merge/2024.5981/Araxis.Merge.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 70.30.5981
```
Please update file name and path to match this change.

File: manifests/a/Araxis/Merge/2024.5981/Araxis.Merge.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Araxis.Merge
+PackageVersion: "2024.5981"
```
Comment (stephengillie) [inline manifests/a/Araxis/Merge/2024.5981/Araxis.Merge.yaml:5]:
```suggestion
PackageVersion: 70.30.5981
```
Please update file name and path to match this change.

**Review by stephengillie — DISMISSED** [review]
Hi @johnburnett,

I believe the `PackageVersion` should match what's displayed by the Apps and features Settings page, if possible. Otherwise a `DisplayVersion` might be necessary - to ensure that the package manager can match this manifest to the information in the Registry.

**Comment by johnburnett** [comment]
> I believe the `PackageVersion` should match what's displayed by the Apps and features Settings page, if possible. Otherwise a `DisplayVersion` might be necessary - to ensure that the package manager can match this manifest to the information in the Registry.

I think I did what you suggested?  (Switched `PackageVersion` to 70.30.5981, and set `DisplayVersion` to be 2024.5981, whichs is the version that shows up in the app UI/on their website).

**Comment by stephengillie** [comment]
> > I believe the `PackageVersion` should match what's displayed by the Apps and features Settings page, if possible. Otherwise a `DisplayVersion` might be necessary - to ensure that the package manager can match this manifest to the information in the Registry.
> 
> I think I did what you suggested? (Switched `PackageVersion` to 70.30.5981, and set `DisplayVersion` to be 2024.5981, whichs is the version that shows up in the app UI/on their website).

Oh - I think those have to be arranged in the opposite way, with the `DisplayVersion` matching the Apps and features page, and the `PackageVersion` matching the UI/website. 

@mdanish-kh @Trenly are we handling this situation in the correct way?

**Comment by mdanish-kh** [comment]
@stephengillie - You're right. It's definitely the other way round. PackageVersion should be used for the marketing version, while the actual / registry version should be added to AppsAndFeaturesEntries > DisplayVersion

https://github.com/microsoft/winget-pkgs/blob/master/doc/FAQ.md#what-is-the-difference-between-a-marketing-version-and-package-version

File: manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Araxis.Merge
+PackageVersion: 70.30.5981
+AppsAndFeaturesEntries:
+- DisplayVersion: "2024.5981"
```
Comment (mdanish-kh) [inline manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.installer.yaml:7]:
```suggestion
AppsAndFeaturesEntries:
- DisplayVersion: 70.30.5981
```

File: manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Araxis.Merge
+PackageVersion: 70.30.5981
```
Comment (mdanish-kh) [inline manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.installer.yaml:5]:
```suggestion
PackageVersion: "2024.5981"
```

File: manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Araxis.Merge
+PackageVersion: 70.30.5981
```
Comment (mdanish-kh) [inline manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.locale.en-US.yaml:5]:
```suggestion
PackageVersion: "2024.5981"
```

File: manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.yaml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Araxis.Merge
+PackageVersion: 70.30.5981
```
Comment (mdanish-kh) [inline manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.yaml:5]:
```suggestion
PackageVersion: "2024.5981"
```

File: manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+PackageLocale: en-US
+Publisher: Araxis Ltd.
```
Comment (mdanish-kh) [inline manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.locale.en-US.yaml:7]:
```suggestion
Publisher: Araxis
```

Should match Control Panel / Apps & Features for better package correlation.

Or you can retain `Araxis Ltd.` here to show this with `winget show` command but add `Publisher: Araxis` under Apps&FeaturesEntries in the installer yaml for providing exact match with Registry entry.

File: manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+LicenseUrl: https://www.araxis.com/merge/windows/licence-agreement.en
+Copyright: Copyright (c) Araxis Ltd.
+CopyrightUrl: https://www.araxis.com/legal
+ShortDescription: Advanced 2 and 3-way file comparison (diff), merging, and folder synchronization
+# Description:
+# Moniker:
+Tags:
+- diff
+- merge
+ReleaseNotes: https://www.araxis.com/merge/windows/release-notes.en
```
Comment (mdanish-kh) [inline manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.locale.en-US.yaml:24]:
```suggestion
# ReleaseNotes:
```

Either include the whole text from ReleaseNotesUrl here or comment out this field as URL is not the "release notes".

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/a/Araxis/Merge` should be renamed from `70.30.5981` -> `2024.5981`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Comment by johnburnett** [comment]
Ok, switch back to the original versioning (but with added `DisplayVersion`), and addressed other feedback.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**